### ISSUE TITLE
Metallurgy Classic Machines

### ISSUE THREAD

**Issue by SkySom** [BrassGoggledCoders/MoarCarts#14]
- [ ] Metal Chests
- [ ] Metal Furnaces (Either Vanilla type or actual furnaces)
- [ ] Smelter
- [ ] Crushers
- [ ] Abstractors

**Comment by RCXcrafter** [comment]
I was just about to suggest this, good to see that you're planning on adding it sometime.

**Comment by SkySom** [comment]
Yep. I'm doing at least the chests for sure, but there's potential for the other.
